src/test/java/Problem2Test.java

```diff
@@ -8,7 +8,27 @@
     // Okay to change
     @Test
     public void testBubbleSort() {
-        assertTrue(false);
+
         // add your tests here
+
+        //test 1
+        int input[] = {3,2,1,5,6};
+        int answer[] = {1,2,3,5,6};
+
+        Problem2.bubbleSort(input);
+        assertArrayEquals(answer, input);
+
+        //test 2
+        int input1[] = {3};
+        int answer1[] = {3};
+
+        Problem2.bubbleSort(input1);
+        assertArrayEquals(answer1, input1);
+
+        //test 3
+        int input0[] = {0};
+
+        Problem2.bubbleSort(input0);
+
```

Comment:
great tests.

let's try to add more tests especially for edge cases such as empty array, null array, etc.

also let's use a loop to go through all the tests without having to repeatedly write the code to call sort and assert.